tests/components/velbus/conftest.py

```diff
@@ -95,6 +97,23 @@ def mock_relay() -> AsyncMock:
     return channel
 
 
+@pytest.fixture
+def mock_select() -> AsyncMock:
+    """Mock a successful velbus channel."""
+    channel = AsyncMock(spec=SelectedProgram)
+    channel.get_categories.return_value = ["select"]
+    channel.get_name.return_value = "select"
+    channel.get_module_address.return_value = 55
+    channel.get_channel_number.return_value = 33
+    channel.get_module_type_name.return_value = "VMB4RYNO"
+    channel.get_full_name.return_value = "Full module name"
+    channel.get_module_sw_version.return_value = "1.1.1"
+    channel.get_module_serial.return_value = "qwerty1234567"
+    channel.get_options.return_value = ["none", "summer", "winter", "holiday"]
```

Comment:
Can these value be any value? Or is this a subset of a fixed set of values?

Reply:
this can be any value

get_gull_name = user defined
module_sw_version = a string representing the firmware version of the module
module_serial = a serial number of the modules
get_options = a fixed set of options, limited to the ones provided